YogaKit/YogaKit.podspec

```diff
@@ -0,0 +1,21 @@
+Pod::Spec.new do |spec|
+  spec.name                 = 'YogaKit'
+  spec.version              = 'v2016.12.26.00'
+  spec.license              = { :type => 'BSD-3-Clause', :file => '../LICENSE' }
+  spec.homepage             = 'https://facebook.github.io/yoga/'
+  spec.authors              = {
+                              'Emil Sjölander'      => 'sjolander.emil@gmail.com',
+                              'Dustin Shahidehpour' => 'email@email.com'
+                            }
+  spec.summary              = 'Yoga is a cross-platform layout engine which implements Flexbox.'
```

Comment:
YogaKit is a view layout library which is powered by Yoga, a layout engine which implements Flexbox.